solr_helpers/__init__.py

```diff
@@ -29,6 +29,18 @@
 # Combined query and result formatter method
 # optionally will normalize facet counting so the response structure is the same for facets+docs and just facets
 def query_solr_and_format_result(query_settings, normalize_facets=True, normalize_groups=True):
+
```

Comment:
Can we get these removed in favor of a method of some sort which is called in the case of a test run?

Reply:
Not sure what you are after? Functions higher up that are tested will still ultimately be calling `query_solr_and_format_result`, where we will have set up the mocking underneath it to return what is needed for that test.

Reply:
Sorry, I mean the prints below this comment, not this method. It'd be better if it was eg.:

if settings.TEST_RUN:
   print_args(args)

and then in settings_unit_test.py we add TEST_RUN = True

Reply:
Oh, got it. Yeah, though it would probably be best named "CAPTURE_FOR_TESTS". This is really only needed because this is being done retroactively, and I don't know what inputs to expect. I can ditch it entirely once the tests are fleshed out.